pkg/flare/envvars.go

```diff
@@ -87,6 +87,11 @@ var allowedEnvvarNames = []string{
 
 	// CI
 	"DD_INSIDE_CI",
+
+	// Cluster agent
+	"DD_CHART_RELEASE_NAME",
+	"DD_AGENT_DAEMONSET",
+	"DD_CLUSTER_AGENT_DEPLOYMENT",
```

Comment:
you can use the "const string" here

```suggestion

	HELM_CHART_RELEASE_NAME ,
	HELM_CHART_RELEASE_NAMESPACE,
	HELM_AGENT_DAEMONSET,
	HELM_CLUSTER_AGENT_DEPLOYMENT,
```